spdx-utils/src/main/kotlin/SpdxConstants.kt

```diff
@@ -35,5 +35,8 @@ object SpdxConstants {
      */
     const val NOASSERTION = "NOASSERTION"
 
-    internal fun isNotPresent(value: String) = value in setOf(NONE, NOASSERTION)
+    internal fun isNotPresent(value: String) = value in setOf(
+        NONE,
+        NOASSERTION
+    )
```

Comment:
Unrelated code change (that also does not seem to be necessary / improve readability anyway).

Reply:
Done.